draft-cardwell-ccwg-bbr.xml

```diff
@@ -173,7 +173,7 @@ Startup mode for BBR.pacing_gain.</t>
 Drain mode, to attempt to drain the estimated queue at the bottleneck link in
 one round-trip or less. As noted in <xref target="BBRDrainPacingGain"/>, any
 value at or below 1 / BBRStartupCwndGain = 1 / 2 = 0.5 will theoretically
-achieve this. BBR uses the value 0.35, which has been shown to offer good
+achieve this. BBR uses the value 0.5, which has been shown to offer good
```

Comment:
Looks like this 0.35 value occurs a second time, lower in the text:

    It uses a pacing_gain of BBRDrainPacingGain = 0.35

Let's change that one as well. Perhaps we should remove the discussion of the exactly value in that other spot, so we don't have this kind of potential disagreement in the text in the future. How about:

old:

      It uses a pacing_gain of BBRDrainPacingGain = 0.35, chosen...

proposed:

      It uses a pacing_gain of BBRDrainPacingGain, with a value chosen ...